tokio-trace-fmt/src/filter.rs

```diff
@@ -298,6 +306,42 @@ where
 #[cfg(test)]
 mod tests {
     use super::*;
+    use span::*;
+    use tokio_trace_core::*;
+
+    struct Cs;
+
+    impl Callsite for Cs {
+        fn add_interest(&self, _interest: Interest) {}
+        fn clear_interest(&self) {}
+        fn metadata(&self) -> &Metadata {
+            unimplemented!()
+        }
+    }
+
+    #[test]
+    fn callsite_enabled_no_span_directive() {
+        let dirs = parse_directives("app=debug");
+        let filter = EnvFilter::new(dirs);
+        let store = Store::with_capacity(1);
+        let ctx = Context::new(&store);
```

Comment:
This isn't a criticism of your branch, just a thought:

Having to make a new empty `Store` and `Context` are unfortunate side-effects of the current `filter` API design, as the `callsite_enabled` function really shouldn't need to be passed a `Context`, but it does so that it can default to calling `self.enabled()` which _does_ take a `Context`. Maybe I should rethink this design...